models/profile-data.json

```diff
@@ -6,15 +6,16 @@
   "properties": {
     "id": {
       "type": "string",
-      "index": true,
-      "id": true
+      "required": true,
```

Comment:
I think only required changed, but the order of the other two fields was switched.